actioncable/lib/action_cable/subscription_adapter/postgresql.rb

```diff
@@ -115,6 +122,11 @@ def shutdown
             Thread.pass while @thread.alive?
           end
 
+          def broadcast(channel, message)
+            message = @adapter.with_broadcast_connection { |pg_conn| pg_conn.unescape_bytea(message) }
```

Comment:
Oh, we don't have a read-to-use connection reference. Let's bring back the decoder then and avoid checking out a real AR connections from the pool.

Sorry for confusion.

Reply:
So we should decrease the compressor threshold by going back to how it was before? What do you think might be a suitable value?

Reply:
There is a simpler way to reuse the connection, I just did it

Reply:
Awesome!